frontend/src/features/rollout-announcement/assets/LogicOrderSvgr.tsx

```diff
@@ -0,0 +1,285 @@
+import { forwardRef, memo, SVGProps } from 'react'
+import { chakra } from '@chakra-ui/react'
+
+const MemoLogicOrderSvgr = memo(
+  forwardRef<SVGSVGElement, SVGProps<SVGSVGElement>>((props, ref) => (
+    <svg
+      width="96"
+      height="96"
+      viewBox="0 0 96 96"
+      fill="none"
+      ref={ref}
+      {...props}
+    >
+      <mask
+        id="mask0_8319_115988"
+        maskUnits="userSpaceOnUse"
+        x="0"
+        y="0"
+        width="96"
+        height="96"
+      >
+        <rect width="96" height="96" fill="#F6F7FC" />
+      </mask>
+      <g>
+        <rect
+          width="198"
+          height="146"
+          transform="translate(-23 -22)"
+          fill="#F6F7FC"
+        />
+        <rect
+          x="84.25"
+          y="83.4384"
+          width="5.503"
+          height="46.9589"
+          rx="2.7515"
+          transform="rotate(-90 84.25 83.4384)"
+          fill="#E4E7F6"
+        />
+        <path
+          d="M101.953 13H50.3878C48.3269 13 46.6562 14.6707 46.6562 16.7316V101.268C46.6562 103.329 48.3269 105 50.3878 105H101.953C104.013 105 105.684 103.329 105.684 101.268V16.7316C105.684 14.6707 104.013 13 101.953 13Z"
+          fill="white"
+          stroke="black"
+          stroke-width="0.5"
```

Comment:
this will output errors, please fix before merging. you need to run the original svg through https://react-svgr.com/playground/, not just wrap it in a react component haha. that's why its called svgr. same for the other svgs.

Reply:
I see.. okay updated